packages/terra-abstract-modal/tests/wdio/abstract-modal-spec.js

```diff
@@ -130,6 +130,7 @@ Terra.describeViewports('Abstract Modal', ['medium'], () => {
       });
 
       it('shifts focus backwards away from the modal dialog', () => {
+        browser.keys(['Shift', 'Tab']);
```

Comment:
is this change because of rect-intl upgrade ..??

Reply:
Not sure could also be due to the chrome driver update for wdio

Reply:
It's better to ensure the cause for these changes. as there are couple of other wdio test specs are also updated.